src/OSPSuite.Core/Domain/Builder/ExpressionProfileBuildingBlock.cs

```diff
@@ -0,0 +1,51 @@
+﻿using OSPSuite.Core.Domain.Services;
+using static OSPSuite.Core.Domain.Constants.ContainerName;
+
+namespace OSPSuite.Core.Domain.Builder
+{
+   public class ExpressionProfileBuildingBlock : StartValueBuildingBlock<ExpressionParameter>
+   {
+      public override string Icon => Type.IconName;
+
+      public virtual string MoleculeName { get; private set; }
+
+      public string Species { get; private set; }
+
+      public ExpressionType Type { set; get; }
+
+      public int PKSimVersion { set; get; }
+
+      public virtual string Category { get; private set; }
+
+      public override string Name
```

Comment:
Looking at the maching PKSIm implementation , events are missing here
You need to throw ent OnPRopertyChangedEvent for Name